### DISCUSSION TITLE
cefsharp support chrome extension api?

### DISCUSSION THREAD

**Discussion by mlzboy** [cefsharp/CefSharp#4513] [General]
Please ask your questions under the Discussions section here on GitHub 
- https://github.com/cefsharp/CefSharp/discussions

---

`CefSharp` is a wrapper around the [Chromium Embedded Framework](https://bitbucket.org/chromiumembedded/cef/overview) in a lot of cases a feature must be added to `CEF` first before it can be used in `CefSharp`. `CEF` has its own `Feature Request Forum` at https://magpcss.org/ceforum/viewforum.php?f=7

--- 

**Describe the solution you'd like**
A clear and concise description of the feature you are proposing

**Describe alternatives you've considered**

**Additional context**

**Checklist:**
- [ ] I have reviewed the [CEF API](https://magpcss.org/ceforum/apidocs3/index-all.html) and have confirmed the feature I'm requesting is possible.
- [ ] The Feature I'm requesting is an improvement to Async Javascript Binding (No new feature are being added to the Sync Javascript binding)
- [ ] An open `PR` exists on the [CEF Pull Requests](https://bitbucket.org/chromiumembedded/cef/pull-requests/) and I'd like to propose this feature is added to `CefSharp` when/if it's merged.

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related questions. Feature requests aren't for asking questions.

Extensions are supported via the [Chrome Runtime](https://github.com/cefsharp/CefSharp/discussions/4123)
